fairlearn/reductions/_moments/bounded_group_loss.py

```diff
@@ -34,12 +35,17 @@ def default_objective(self):
         """Return a default objective."""
         return MeanLoss(self.reduction_loss)
 
-    def load_data(self, X, y, **kwargs):
+    def load_data(self, X, y, *, sensitive_features):
         """Load data into the moment object."""
-        kwargs_mod = kwargs.copy()
+        X_train, y_train, sf_train, _ = \
```

Comment:
```suggestion
       _, _, sf_train, _ = \
```

Reply:
Why? The Moments don't call anything else, and the code in them has always assumed that they are dealing with a Series for `y` (hence the `y_reformat` property which I added).

Reply:
For example, our `ConditionalLossMoment` should very easily work with multi-class classification. In that case it's often useful to represent `y` as a two-dimensional array.

For the time being, as I suggest below, we can keep `y_train` here and rename the property to something like `_y_reformat` until we figure out whether we really want to export it. My hunch is that we shouldn't export it and we just currently have some abstraction leakage [to be fixed in a future PR].

Reply:
I've renamed `y_reformat` to `_y_as_series`. Not that I'm a huge fan of objects accessing each other's innards either. But then again, 'private' is merely a convention in Python.